study.md

```diff
@@ -95,11 +98,11 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+<!-- Seed data is a clean way to set up a databse for a new app -->
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+<!-- Yes this would be a good method of debuging problems. -->
 ```
```

Comment:
I'm not going to give a hard no here but creating dummy data in you development db would probably be a better idea as Jeff corrected me during my lesson.